tests/test_mibs.py

```diff
@@ -30,3 +35,4 @@ def test_init_sync_d_lag_tables(self):
 
         self.assertTrue("PortChannel_Temp" in lag_name_if_name_map)
         self.assertTrue(lag_name_if_name_map["PortChannel_Temp"] == [])
+        self.assertTrue(lag_sai_map["PortChannel01"] == "2000000000006")
```

Comment:
What is "2000000000006"?

Reply:
It was added in https://github.com/Azure/sonic-snmpagent/pull/169.

Reply:
Resolved